src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -444,7 +443,9 @@ export class ComponentTestUtils<T extends AbstractGameComponent, P extends Compa
         }
         expect(this.canUserPlaySpy).toHaveBeenCalledOnceWith(nameInFunction);
         this.canUserPlaySpy.calls.reset();
-        expect(this.chooseMoveSpy).not.toHaveBeenCalled();
+        expect(this.chooseMoveSpy)
+            .withContext('chooseMove has been called, use expectMoveFailure in the test if the code is correct, or cancelMove in the coe if the test is correct.')
```

Comment:
coe -> code
Also, pas la meilleure formulation je dirais, plutôt "use expectMoveFailure if we want the code to fail, or cancelMove if ???". Car le code ET le test sont TOUJOURS correct, sinon c'est un bug à corriger !

Reply:
Bah, quand tu vois ça soit ton code est pas correct soit ton test est pas correct et oui c'est à corriger, doooonc ?

Reply:
ah, alors *was* plutôt que *is* !

Reply:
Mais pourquoi au passé ?

Reply:
parce qu'on souhaite fixer le code/test, pour que les deux soient corrects! Donc ça devrait devenir un état du passé ! (Puis en l'état j'avais pas compris cette phrase, mais avec un *was*, j'aurais mieux compris)